lightning-background-processor/src/lib.rs

```diff
@@ -773,7 +796,10 @@ impl BackgroundProcessor {
 					handle_network_graph_update(network_graph, &event)
 				}
 				if let Some(ref scorer) = scorer {
-					if update_scorer(scorer, &event) {
+					use std::time::SystemTime;
```

Comment:
since this functions uses system time now, it should probably be `#[cfg(all(feature = "std"), not(feature = "no-std")]` to handle when you'll be able to use both flags together in the future